pkg/controller/installation/windows_controller.go

```diff
@@ -0,0 +1,455 @@
+	watches              map[runtime.Object]struct{}
+	autoDetectedProvider operatorv1.Provider
+	status               status.StatusManager
+	enterpriseCRDsExist  bool
+	amazonCRDExists      bool
+	clusterDomain        string
+}
+
+// newWindowsReconciler returns a new reconcile.Reconciler
+func newWindowsReconciler(mgr manager.Manager, opts options.AddOptions) (*ReconcileWindows, error) {
+	statusManager := status.New(mgr.GetClient(), "calico-windows", opts.KubernetesVersion)
+
+	r := &ReconcileWindows{
+		config:               mgr.GetConfig(),
+		client:               mgr.GetClient(),
+		scheme:               mgr.GetScheme(),
+		watches:              make(map[runtime.Object]struct{}),
+		autoDetectedProvider: opts.DetectedProvider,
+		status:               statusManager,
+		amazonCRDExists:      opts.AmazonCRDExists,
+		enterpriseCRDsExist:  opts.EnterpriseCRDExists,
+		clusterDomain:        opts.ClusterDomain,
+	}
+	r.status.Run(opts.ShutdownContext)
+	return r, nil
+}
+
+// Reconcile reads that state of the cluster for a Installation object and makes changes based on the state read
+// and what is in the Installation.Spec. The Controller will requeue the Request to be processed again if the returned error is non-nil or
+// Result.Requeue is true, otherwise upon completion it will remove the work from the queue.
+func (r *ReconcileWindows) Reconcile(ctx context.Context, request reconcile.Request) (reconcile.Result, error) {
+	reqLogger := logw.WithValues("Request.Namespace", request.Namespace, "Request.Name", request.Name)
+	reqLogger.Info("Reconciling Installation.operator.tigera.io")
+
+	// Get the installation object if it exists so that we can save the original
+	// status before we merge/fill that object with other values.
+	instance := &operatorv1.Installation{}
+	if err := r.client.Get(ctx, utils.DefaultInstanceKey, instance); err != nil {
+		if apierrors.IsNotFound(err) {
+			reqLogger.Info("Installation config not found")
+			r.status.OnCRNotFound()
+			return reconcile.Result{}, nil
+		}
+		reqLogger.Error(err, "An error occurred when querying the Installation resource")
+		return reconcile.Result{}, err
+	}
+
+	// Don't render calico-node-windows if it's disabled in the installation
+	if !common.WindowsEnabled(instance.Spec) {
+		reqLogger.Info("Calico Windows daemonset is disabled in the operator installation")
+		return reconcile.Result{}, nil
+	}
+
+	// Mark CR found so we can report converter problems via tigerastatus
+	r.status.OnCRFound()
+	// FIXME: add logic to merge Installation status metadata
+
+	// Changes for updating Installation status conditions.
+	if request.Name == InstallationName && request.Namespace == "" {
+		ts := &operatorv1.TigeraStatus{}
+		err := r.client.Get(ctx, types.NamespacedName{Name: InstallationName}, ts)
+		if err != nil {
+			return reconcile.Result{}, err
+		}
+		instance.Status.Conditions = status.UpdateStatusCondition(instance.Status.Conditions, ts.Status.Conditions)
```

Comment:
I think we still need to get rid of this block for now - it's going to fight with the same code from core_controller.go